quarkus-integration/quarkus/deployment/src/test/java/ai/timefold/solver/quarkus/TimefoldProcessorMultipleSolversInvalidSolutionClassTest.java

```diff
@@ -78,6 +82,14 @@ class TimefoldProcessorMultipleSolversInvalidSolutionClassTest {
                     .hasMessageContaining(
                             "Unused classes ([ai.timefold.solver.quarkus.testdata.chained.domain.TestdataChainedQuarkusSolution]) found with a @PlanningSolution annotation."));
 
+    @Inject
+    @Named("solver1")
+    SolverManager<?, ?> solverManager1;
+
+    @Inject
+    @Named("solver2")
+    SolverManager<?, ?> solverManager2;
```

Comment:
Why do we need to add these, if the test worked without them before?
Smells of a backwards incompatible change.

Reply:
If you are using named solver managers, you should have these annotations somewhere in your code base. I argue the old behaviour is a bug. And all bugs are incompatible changes. As noted in my commit:

"""
As a consequence, there is a behaviour change in the very contrived
case of "exactly one named solver in properties (no default), no Named annotations,
injected default solver". Since there was no Named annotations,
the (empty) default solver properties will be checked instead of the single
named solver properties. I consider the old behaviour to be a bug
(named properties should not affect default properties),
but the tests expect that behaviour.
"""

Reply:
(there is also a question if we should add a fail fast if there a named build time solver without a corresponding Named annotation (ex: misspelled the name by accident, "solve1" instead of "solver1"))

Reply:
I think that adding the fail-fast makes sense. We can give a nice exception message, listing all the named solvers present in config which do not have the accompanying annotation.